app/src/test/java/com/github/dedis/student20_pop/utility/security/HashTest.java

```diff
@@ -2,18 +2,27 @@
 
 import org.junit.Test;
 
+import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNotNull;
 import static org.junit.Assert.assertThrows;
 
 public class HashTest {
 
     @Test
     public void hashNullDataTest() {
-        assertThrows(IllegalArgumentException.class, () -> Hash.hash(null));
+        assertThrows(IllegalArgumentException.class, () -> Hash.hash((String) null));
     }
 
     @Test
     public void hashTest() {
         assertNotNull(Hash.hash("Data to hash"));
     }
+
+    // Hashing : test 0 \fwa"fwa-fwa
+    // Expected : ["test","0","\\fwa\"fwa-fwa"]
+
+    @Test
+    public void objectsHashingWorks() {
```

Comment:
I would suggest to rename the method `hashObjectsTest` to follow the conventions. Also the comments would be better inside the method.

Reply:
Done !